From 267bdee63c4c2d1fc95b03de8b6198be6594d9cb Mon Sep 17 00:00:00 2001
From: Nikita Yushchenko <nikita.yoush@cogentembedded.com>
Date: Sun, 16 Jun 2024 16:23:52 +0500
Subject: [PATCH 053/157] arm64: configs: vc4: enable what is needed for
 nfsroot over ravb

This includes ravb dirver, micrel phy driver for use with KSZ8061
connected to ravb, nfs, and nfsroot.

Signed-off-by: Nikita Yushchenko <nikita.yoush@cogentembedded.com>

diff --git a/arch/arm64/configs/renesas-vc4_defconfig b/arch/arm64/configs/renesas-vc4_defconfig
index ae5c672fef85..159552fbbf87 100644
--- a/arch/arm64/configs/renesas-vc4_defconfig
+++ b/arch/arm64/configs/renesas-vc4_defconfig
@@ -288,6 +288,7 @@ CONFIG_VETH=m
 # CONFIG_NET_VENDOR_QUALCOMM is not set
 # CONFIG_NET_VENDOR_RDC is not set
 # CONFIG_NET_VENDOR_REALTEK is not set
+CONFIG_RAVB=y
 # CONFIG_NET_VENDOR_ROCKER is not set
 # CONFIG_NET_VENDOR_SAMSUNG is not set
 # CONFIG_NET_VENDOR_SEEQ is not set
@@ -304,6 +305,7 @@ CONFIG_VETH=m
 # CONFIG_NET_VENDOR_VIA is not set
 # CONFIG_NET_VENDOR_WIZNET is not set
 # CONFIG_NET_VENDOR_XILINX is not set
+CONFIG_MICREL_PHY=y
 # CONFIG_WLAN is not set
 CONFIG_NET_FAILOVER=y
 CONFIG_INPUT_FF_MEMLESS=y
@@ -393,7 +395,10 @@ CONFIG_EXFAT_FS=m
 CONFIG_TMPFS=y
 CONFIG_HUGETLBFS=y
 # CONFIG_MISC_FILESYSTEMS is not set
-# CONFIG_NETWORK_FILESYSTEMS is not set
+CONFIG_NFS_FS=y
+# CONFIG_NFS_V2 is not set
+CONFIG_ROOT_NFS=y
+# CONFIG_RPCSEC_GSS_KRB5 is not set
 CONFIG_NLS=y
 CONFIG_NLS_CODEPAGE_437=y
 CONFIG_NLS_ISO8859_1=y
-- 
2.25.1

